tls.go

```diff
@@ -7,6 +7,7 @@ import (
 	"io/ioutil"
 )
 
+// TLS13Config returns *tls.Config with TLS 1.3 settings.
```

Comment:
The returned TLS configuration supports ONLY TLS 1.3 and does not enforce client side authentication.